docs/development.txt

```diff
@@ -0,0 +1,84 @@
+[[development-process]]
+JanusGraph Development Process
+==============================
```

Comment:
Please use Asciidoc best practices for section titles, as described in [their docs](http://asciidoctor.org/docs/asciidoc-recommended-practices/#section-titles).

Specifically, this line should be:

> = JanusGraph Development Process

and other section headings should use the number of leading `=` equal to their header level. 

The current approach of using different types of characters on a line is complex (need to remember which character means which heading level) and error-prone (must equal in length to the heading, or else it breaks. This is the type of thing I had to deal with during the global `s/Titan/Janusgraph/` cleanup; see [this commit](https://github.com/JanusGraph/janusgraph/commit/e21734fd1ffef90ef44a0ad7d2da0b2f5859baa4) for more details.

I am working on a change to convert all existing headers to the Asciidoc-recommended approach, so it would be nice to not add any more of these anachronistic section titles.